Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -72,6 +72,7 @@
 
   private boolean mDeviceHasTouchScreen;
   private boolean mMenuVisible;
+  private static boolean sStartingEmulation = false;
```

Comment:
This variable should probably be renamed because its name is a bit misleading. It is set to `true` when the emulation is launching and/or running.